Readme.md

```diff
@@ -144,6 +144,36 @@ gen:    i don' t know .
 ......
 ```
 
+## Experiment
+
+Based on a smaller version of `OpenSubtitles`, we did the following experiments.
+
+| encoder | decoder | batchnorm | learning rate | droprate | decode mode | dev perplexity | test perplexity |
+| :-----: | :-----: | :-------: | :-----------: | :------: | :---------: | :------------: | :-------------: |
+|   128   |   128   |     ×     |    0.0001     |    0     |   samplek   |     95.207     |     100.676     |
+|   128   |   128   |     ×     |    0.0001     |   0.2    |   samplek   |     93.559     |     99.287      |
+|   128   |   128   |     √     |    0.0001     |    0     |   samplek   |    105.649     |     112.818     |
+|   128   |   128   |     √     |    0.0001     |   0.2    |   samplek   |     95.894     |     102.243     |
+|   150   |   150   |     ×     |    0.0001     |   0.2    |   samplek   |     90.153     |     95.072      |
+|   175   |   175   |     ×     |    0.0001     |   0.2    |   samplek   |     88.971     |     94.698      |
+|   256   |   256   |     ×     |    0.0001     |   0.2    |   samplek   |     89.374     |     94.272      |
+|   175   |   175   |     ×     |    0.0001     |   0.2    |    beam     |     89.253     |     93.674      |
+|   175   |   175   |     √     |    0.0001     |   0.2    |   samplek   |     97.704     |     102.851     |
+|   175   |   175   |     ×     |    0.0001     |   0.1    |   samplek   |     90.149     |     95.310      |
+|   175   |   175   |     ×     |    0.0001     |   0.3    |   samplek   |     89.750     |     95.042      |
+|   175   |   175   |     ×     |    0.0005     |   0.2    |   samplek   |     88.688     |     93.421      |
```

Comment:
Reorganize the table, with the best on the top.   Use **bold** to mark the difference between the best and this line

Reply:
Give your command to reproduce the best experiment